agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -16,22 +16,32 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 # Defaults
 group=default
 dir="/tmp"
-
-# always notify the user, as collection can sometimes take a while
-echo "Collecting system information"
+sysinfo_opts_default=( block libvirt kernel_config sos topology )
+sysinfo_opts=${sysinfo_opts_default[*]}
+check=false
+
+# Display sysinfo options 
+function display_sysinfo_opts {
+	printf -- "\t\t       --sysinfo=str,          str = comma seperated values of sysinfo to be collected\n"
```

Comment:
Notice that `pbench-user-benchmark`'s usage information is emitted with two leading tabs, but `pbench-collect-sysinfo` is emitted with only one leading tab.

So it does not seem worth to try to modify this method to print usage information in full.  Perhaps we modify `display_sysinfo_opts` to only emit the list of options, and just embed that in each tool with the proper formatting?

What do you think?